app/components/FormFields/validate.js

```diff
@@ -12,6 +12,12 @@ const byteLength = val => encodeURI(val).split(/%..|./).length - 1;
 const maxByteLength = val =>
   byteLength(val) > 256 ? messages.wrongByteLength : undefined;
 
+const maxBountyDaysLength = val =>
```

Comment:
Try add validation functions that can be reused instead of adding a new one for each case
Add `valueHasToBeMoreThan` validation instead.